.github/workflows/sycl-nightly.yml

```diff
@@ -108,6 +108,13 @@ jobs:
             runner: '["Linux", "arc"]'
             image_options: -u 1001
             target_devices: opencl:cpu
+
+          - name: Preview mode on SPR/PVC
```

Comment:
```suggestion
          - name: Preview mode on PVC + L0
```

Reply:
Hmm, not sure about that. Initially, I wanted to target everything but that uncovered some gap in our testing. I'm addressing that gap separately and plan on bringing additional targets once that done.